src/java.xml.crypto/share/classes/com/sun/org/apache/xml/internal/security/algorithms/implementations/SignatureECDSA.java

```diff
@@ -54,21 +55,25 @@
     /** Field algorithm */
     private Signature signatureAlgorithm;
 
+    /** Length for each integer in signature */
+    private int signIntLen = -1;
+
     /**
      * Converts an ASN.1 ECDSA value to a XML Signature ECDSA Value.
      *
      * The JAVA JCE ECDSA Signature algorithm creates ASN.1 encoded (r, s) value
      * pairs; the XML Signature requires the core BigInteger values.
      *
      * @param asn1Bytes
+     * @param rawLen
```

Comment:
Well, it looks like a "style" to only list the param names without any explanation in this file and nearby files. Since the body of this method is only one line I assume people curious about the meaning of the parameters can just navigate to `ECDSAUtils.convertASN1toXMLDSIG` to read the description there.

Can we just keep it "clean"?

Reply:
Ok, sounds reasonable.